### PR TITLE
[MM-55101] Replace usage of LocalizedIcon in 'multiselect.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by mozi47**
#### Summary
Replace LocalizedIcon with i tag in multiselect.tsx

#### Ticket Link
Fixes: #25109

https://mattermost.atlassian.net/browse/MM-55101

### CODE REVIEW

**Comment by mozi47** [comment]
I couldn't get where this component comes in the app, which is why the screenshot was not added. If you can help me with that will be great.

**Comment by M-ZubairAhmed** [comment]
@mozi47 can you please take a look and follow the guidelines mentioned https://github.com/mattermost/mattermost/issues/25109#issue-1959027425

File: webapp/channels/src/components/multiselect/multiselect.tsx [hunk 1]
```diff
@@ -332,9 +332,9 @@ export default class MultiSelect<T extends Value> extends React.PureComponent<Pr
             noteTextContainer = (
                 <div className='multi-select__note'>
                     <div className='note__icon'>
-                        <LocalizedIcon
+                        <i
                             className='fa fa-info'
-                            title={{id: 'generic_icons.info', defaultMessage: 'Info Icon'}}
+                            title={this.props.intl.formatMessage({id: 'generic_icons.info', defaultMessage: 'Info Icon'})}
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/multiselect/multiselect.tsx:337]: this component doesnt have props `intl` injected how did this work?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Please address the CI failures

**Comment by M-ZubairAhmed** [comment]
@mozi47 it still has error can you please check , we can only begin the review once the CI is green 😄

**Comment by M-ZubairAhmed** [comment]
@mozi47 a lot of failures are in test file of the component, you can try looking at other components where injectIntl is used and how there test files are

**Comment by mozi47** [comment]
I need help. Updation of snapshto is not happening as expected!

**Comment by M-ZubairAhmed** [comment]
> I need help. Updation of snapshto is not happening as expected!

Thats fine, i will update from my end. Can you check the failing CIs

**Comment by M-ZubairAhmed** [comment]
> I need help. Updation of snapshto is not happening as expected!

You need to run `npm run test -- -u` from channels folder

**Comment by M-ZubairAhmed** [comment]
@mozi47 please check the failing check types. And for the failing test i would recommend taking a look at other tests which uses the injectIntl

**Pull request closed without merging**